### PR TITLE
Gentle slimecross extracts

### PR DESCRIPTION
**Pull request by Dingo-Dongler**
## About The Pull Request

Adds pink slime crossbreeding, gentle extracts. They are able to be used like luminescent activations, with the same cooldowns.
Completes issue https://github.com/BeeStation/BeeStation-Hornet/issues/1546. Some activations may need to be changed.
Also, orange slime extract major extract doesn't work with regular luminescent activation so I'll try fix that.

## Why It's Good For The Game

Adds some extra fun options for xenobiology.

## Changelog
:cl:
add: Gentle slime crossbreeds can be made with pink extracts. They can be used to emulate luminescent activations of extracts.
/:cl:

### CODE REVIEW

File: code/modules/research/xenobiology/crossbreeding/gentle.dm [hunk 1]
```diff
@@ -0,0 +1,139 @@
+/obj/item/slimecross/gentle
+	name = "gentle extract"
+	desc = "It pulses slowly, as if breathing."
+	effect = "gentle"
+	effect_desc = "Use to activate minor effect, Alt-click to activate major effect."
+	icon_state = "gentle"
+	var/extract_type
+	var/obj/item/slime_extract/extract
+	var/cooldown = 5 //This is in seconds
```
Comment (zeskorion) [inline code/modules/research/xenobiology/crossbreeding/gentle.dm:9]: Make this cooldown a great deal higher, and i'll merge it

**Review by zeskorion — COMMENTED** [review]
A vote for this would be a mere formality, so i wont bother

**Comment by Rukofamicom** [comment]
_holy crap someone finally did one_

File: code/modules/research/xenobiology/crossbreeding/gentle.dm (hunk 1 above)
Comment (Dingo-Dongler) [reply code/modules/research/xenobiology/crossbreeding/gentle.dm:9]: This is the initial cooldown, the cooldown after use is defined by the luminescent cooldown in activations.

**Review by zeskorion — APPROVED** [review]

**Pull request merged**